pkgs/development/libraries/libxcrypt/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation (finalAttrs: {
     "--enable-hashes=${enableHashes}"
     "--enable-obsolete-api=glibc"
     "--disable-failure-tokens"
-  ] ++ lib.optionals (stdenv.hostPlatform.isMusl || stdenv.hostPlatform.libc == "bionic") [
+  ] ++ lib.optionals (stdenv.hostPlatform.isMusl || stdenv.hostPlatform.libc == "bionic" || lib.hasInfix "-march=native" stdenv.cc.NIX_CFLAGS_COMPILE) [
     "--disable-werror"
```

Comment:
IMO this should be made unconditional, Werror's for us are annoyances
 
 @mweinelt

Reply:
Would be easier than accumulating more conditionals.

Reply:
can't push here

https://github.com/NixOS/nixpkgs/pull/241531

Reply:
helsinki-systems :shrug: